source/Score.js

```diff
@@ -0,0 +1,60 @@
+import Reference from "./Reference.js"
+
+
+export default class Score {
+    constructor(playerNumber) {
+        this.count = 0
+        this.playerNumber = (playerNumber === undefined) ? game.playerCount : playerNumber
+        this.domElement = document.createElement("div")
+        this.domElement.className += " player-score"
+        this.domElement.style.position = "absolute"
+        this.domElement.style.textAlign = "center"
+        var whiteText = document.createElement("div")
+        var colorText = document.createElement("div")
+        whiteText.className += " foreground-score"
+        colorText.id = "player-" + this.playerNumber + "-score"
+        colorText.className += " shadow-score"
+        this.domElement.appendChild(whiteText)
+        this.domElement.appendChild(colorText)
+        document.body.appendChild(this.domElement)
+        this.gameCanvas = document.getElementById("gameCanvas")
+        this.update()
+
+    }
+
+    update(delta) {
+        var scoreElementWidth = this.gameCanvas.offsetWidth * 0.995 / game.playerCount
+        var scoreLeftOffset = this.gameCanvas.offsetLeft + scoreElementWidth * (this.playerNumber - 1)
```

Comment:
This is suuuch a pain to calculate (and more importantly, recalculate over and over with each loop). We shouldn't have to maintain the offset like this.

At the moment, the `@include aspect-ratio` is being applied to the `canvas`, but let's instead apply it to a parent `div`, which has 2 children, one is the `canvas`, and the other is the `div`s for the UI. That way, this content will always be positioned absolutely in it's parent class by CSS, not by JS.